src/components/Dossier/Tag.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { css } from 'glamor'
+import { mUp } from '../TeaserFront/mediaQueries'
+import { sansSerifMedium16, sansSerifMedium20 } from '../Typography/styles'
+import FolderOpen from 'react-icons/lib/fa/folder-open'
+
+const styles = {
+  tag: css({
+    display: 'inline-block',
+    ...sansSerifMedium16,
+    margin: '0 0 18px 0',
+    [mUp]: {
+      ...sansSerifMedium20,
+      margin: '0 0 28px 0'
+    }
+  }),
+  icon: css({
+    fontSize: '24px',
```

Comment:
Yes, it works, and I think it's a nice way to implement responsive icon sizes with media queries only. But since we don't need a responsive size here, I've changed things back to the size prop.